### ISSUE TITLE
Change the Ender Dragon's Models and Mechanics to More "Ice and Fire Style"

### ISSUE THREAD

**Issue by Blunder-G** [AlexModGuy/Ice-and-Fire-Suggestions#1140]
Ice and Fire's dragons are breathtaking. Gorgeous models, sometimes buggy but great animations and mechanics... Ender Dragon's models are very grossy and very easy to fight. 

Well, imagine the Ender Dragon is the endgame boss with black scales and powerful enough to be stage 5 or more dragon. A giant creature flying in the sky, with huge wings and purple bones; the strongest dragon in the game that sprays dragon breath rather than throwing it as a ball... Imagine the dragon egg is remodeled and there is a dragon we can breed to ride ONLY AT THE END. Probably one of the next updates will be the End Update and I would love to ride a dragon like this over the land of the End.

But being able to do this will bring an "Ice and Fire 3.0" style update. But I still want to share it because it's actually both a suggestion and a solution to a frustrating problem. The biggest problem of Minecraft is that the bosses are very bad. Neither Datapacks nor other plugins can change this...

**Comment by Aceplante** [comment]
YES, please add this. This seems to be a highly requested feature but its been over 6 years and this still has not been added
